src/device-registry/utils/insert-measurements.js

```diff
@@ -112,4 +112,72 @@ const insert = async (tenant, transformedMeasurements) => {
   }
 };
 
-module.exports = insert;
+const bulkInsert = async (transformedMeasurements) => {
```

Comment:
What is the difference of this method from the existing one of insert?
Aren't we just duplicating code?
@NoahNsimbe

Reply:
The difference is in the method arguments. I have modified the method to call the existing one. @Baalmart